src/render/HTMLRenderer.php

```diff
@@ -28,12 +28,65 @@ protected static function escapeAttribute(string $text): string {
 
   // This is the list from the reference implementation
   const keyset<string> URI_SAFE = keyset[
-    '-', '_', '.', '+', '!', '*', "'", '(', ')', ';', ':', '%', '#', '@', '?',
-    '=', ';', ':', '/', ',', '+', '&', '$',
-    'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j',
-    'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
-    'u', 'v', 'w', 'x', 'y', 'z',
-    '1', '2', '3', '4', '5', '6', '7', '8', '9', '0',
+    '-',
```

Comment:
Could you restore the old formatting and add `//hackfmt-ignore` above the expression?